### PR TITLE
chatroom created

### LINKED ISSUE #42: create a chatroom

### PR DESCRIPTION
**Pull request by Kaylee2319**
closes #42

### CODE REVIEW

File: client/src/components/chat.jsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+  };
+
+  const sendMessage = (e) => {
+    e.preventDefault();
+    socketIo.emit('send message', { author: username, message: message });
+    setMessage('');
+  };
+  let isSentByCurrentUser = false;
+
+  if (currentUser === currentUser) {
```
Comment (ErnieAtLYD) [inline client/src/components/chat.jsx:32]: Wait a sec -  won't `(currentUser === currentUser)` always return true? And if that's the case, all you need is line 33 without lines 30-32. Unless Line 32 is a mistake...

File: client/src/components/chat.jsx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+                className="messagePic"
+                src={
+                  currentUser?.avatar
+                    ? currentUser?.avatar
+                    : 'https://files.willkennedy.dev/wyncode/wyncode.png'
+                }
+                alt="profile"
+              />
+              <p className="theoutput">
+                <strong>{currentUser?.name}Kaylee:</strong>
```
Comment (ErnieAtLYD) [inline client/src/components/chat.jsx:67]: I'm assuming this is test code, right?

File: client/src/css/chat.css [hunk 3]
```diff
@@ -0,0 +1,80 @@
+@import url('https://fonts.googleapis.com/css2?family=Unica+One&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Cabin&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400&display=swap');
```
Comment (ErnieAtLYD) [inline client/src/css/chat.css:3]: You can actually combine lines 1-3: https://stackoverflow.com/a/33984674

**Review by ErnieAtLYD — COMMENTED** [review]
So I'm not crazy - this PR isn't finished, right? Did you know you can make GitHub pull requests into drafts? https://github.blog/2019-02-14-introducing-draft-pull-requests/

File: client/src/components/chat.jsx (hunk 1 above)
Comment (DanRie19) [reply client/src/components/chat.jsx]: Spoke With Kaylee, this is resolved.

File: client/src/components/chat.jsx (hunk 2 above)
Comment (DanRie19) [reply client/src/components/chat.jsx]: Test code, resolved

File: client/src/css/chat.css (hunk 3 above)
Comment (DanRie19) [reply client/src/css/chat.css:3]: Will refactor on next PR.

**Pull request merged**